src/zenml/zen_server/routers/auth_endpoints.py

```diff
@@ -255,6 +258,7 @@ def generate_access_token(
     LOGIN,
     response_model=Union[OAuthTokenResponse, OAuthRedirectResponse],
 )
+@limiter.limit("5/minute;1000/day")
```

Comment:
@avishniakov could you also make these values configurable? I recommend you use the ServerSettings class for this and just add two more configuration options for the 5 and 1000 values.

Reply:
Ah, yeah, I wanted to do this but forgot - thx for the reminder. We talked this through with Alexej yesterday